data/human/deep missions.txt

```diff
@@ -3165,21 +3161,21 @@ mission "Home for Skadenga 2"
 			label fancypants
 			`	You are brought to large research complex just outside the spaceport, and dispoisted in a high-end, white, minimalistic conference room with two well-dressed and important looking men. "Thank you, Lars. Take the guards and wait for us outside," the man seated across from you says.`
 			`	"Of course, sir." The administrator, whose name seems to be Lars, turns and is followed outside by his two goons.`
-			`	The man who spoke looks at you with a friendly smile and unironically says, "I am so glad you agreed to meet with us. I realize an accomplished Captain like yourself is in great demand, so we will endeavor to keep this brief."`
```

Comment:
Seems uneconomical to replace what is expressed with one word, with five.

Reply:
I just felt that only using "unironically" made the text feel a little [rushed, stilted] compared to using the phrase.